pvr.hts/resources/settings.xml

```diff
@@ -10,16 +10,20 @@
 	<setting id="response_timeout" type="enum" label="30007" values="1|2|3|4|5|6|7|8|9|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31|32|33|34|35|36|37|38|39|40|41|42|43|44|45|46|47|48|49|50|51|52|53|54|55|56|57|58|59|60" default="4" />
   </category>
 
-  <category label="30100">
+  <category label="30008">
+    <setting label="30050" type="lsep"/>
+	<setting id="autorec_approxtime" type="bool" label="30051" default="true"/>
```

Comment:
Haha. default="false", please. PhobosK and I are in favour of the current approach, you are in favour of the new approach. 2 : 1 for the current approach. ;-)